src/components/Artwork/SandboxPreview.tsx

```diff
@@ -76,12 +77,12 @@ export const SandboxPreview = forwardRef<ArtworkIframeRef, Props>(
     // the URL of the iframe gets updated whenever ID / hash changes
     useEffect(() => {
       if (iframeRef.current && id !== "0") {
-        const previewUrl = `${location.origin}/sandbox/preview.html?id=${id}&fxhash=${hash}`
+        const previewUrl = `${location.origin}/sandbox/preview.html?id=${id}&fxhash=${hash}&fxparams=${fxparams}`
```

Comment:
what about if it's not a params project? guessing this URL still works fine?

Reply:
👍